### ISSUE TITLE
More tests needed

### ISSUE THREAD

**Issue by tpaviot** [tpaviot/oce#90]
Any contribution is here welcome to increase the number of OCE tests.

**Comment by jf---** [comment]
The scope of this issue is pretty vague... / inactionable... / stalled...
A candidate for defragmenting the issue tracker?

**Comment by thatcadguy** [comment]
While I'd love to reduce the number of open issues more, I think we should leave this open. It's a reminder that we need to be constantly adding to https://github.com/tpaviot/oce/tree/master/test when we find bugs.

**Comment by thatcadguy** [comment]
Really it ought to be part of the workflow every time we merge a non-trivial change

**Comment by tpaviot** [comment]
@thatcadguy I agree that tests are part of the workflow. However, people usually don't contribute tests along with their patches (see for instance PR #547). I really wonder why, maybe because contributors fail at isolating the bug and provide a small and atomic test.
